cabot/settings.py

```diff
@@ -91,6 +93,7 @@
 )
 
 MIDDLEWARE_CLASSES = (
+    'django.middleware.http.ConditionalGetMiddleware',
```

Comment:
-> uwsgi does not provide ContentLength headers, this middleware adds it (among other things)